common/components/PrintableWallet/index.jsx

```diff
@@ -0,0 +1,246 @@
+        height: '150px',
+        backgroundSize: '100%'
+      },
+
+      // Address / private key info
+      infoContainer: {
+        float: 'left',
+        width: '85%',
+        padding: '0 20px'
+      },
+      infoText: {
+        margin: '0 0 8px',
+        textAlign: 'left',
+        fontSize: '14px',
+        fontFamily: 'Menlo, Monaco, Consolas, "Courier New", monospace',
+        fontWeight: 300
+      },
+      infoLabel: {
+        fontWeight: 600
+      },
+
+      identiconContainer: {
+        position: 'absolute',
+        right: '15px',
+        bottom: '45px'
+      },
+      identiconImg: {
+        float: 'left',
+        width: '42px',
+        height: '42px',
+        backgroundSize: '100%',
+        borderRadius: '50%',
+        boxShadow: `
+					inset rgba(255, 255, 255, 0.5) 0 2px 2px,
+					inset rgba(0, 0, 0, 0.6) 0 -1px 8px
+				`
+      },
+      identiconText: {
+        float: 'left',
+        width: '130px',
+        padding: '0 5px',
+        margin: '12px 0 0',
+        fontSize: '9px',
+        textAlign: 'center'
+      }
+    };
+
+    return (
+      <div style={styles.container} ref={el => (this._wallet = el)}>
+        <img src={sidebarImg} style={styles.sidebar} />
+        <img src={ethLogo} style={styles.ethLogo} />
+
+        <div style={styles.block}>
+          <img src={qrCodeAddress} style={styles.qrCode} />
+          <p style={styles.blockText}>YOUR ADDRESS</p>
+        </div>
+
+        <div style={styles.block}>
+          <img src={notesBg} style={styles.qrCode} />
+          <p style={styles.blockText}>AMOUNT / NOTES</p>
+        </div>
+
+        <div style={styles.block}>
+          <img src={qrCodePkey} style={styles.qrCode} />
+          <p style={styles.blockText}>YOUR PRIVATE KEY</p>
+        </div>
+
+        <div style={styles.infoContainer}>
+          <p style={styles.infoText}>
+            <strong style={styles.infoLabel}>Your Address:</strong>
+            <br />
+            {address}
+          </p>
+          <p style={styles.infoText}>
+            <strong style={styles.infoLabel}>Your Private Key:</strong>
+            <br />
+            {privateKey}
+          </p>
+        </div>
+
+        <div style={styles.identiconContainer}>
+          <img src={makeIdenticon(address)} style={styles.identiconImg} />
+          <p style={styles.identiconText}>
+            Always look for this icon when sending to this wallet
+          </p>
+        </div>
+        <div style={styles.identicon} />
+      </div>
+    );
+  }
+
+  render() {
+    const qrCodesReady = this.state.qrCodePkey && this.state.qrCodeAddress;
+    const btnDisabled = qrCodesReady ? '' : 'btn-disabled';
+    const walletContainerStyle = {};
+
+    return (
+      <div>
+        {this._renderPaperWallet()}
+        <a
+          role="button"
+          aria-label={translate('x_Print')}
+          aria-describedby="x_PrintDesc"
+          className={`btn btn-lg btn-primary ${btnDisabled}`}
```

Comment:
I think adding a check in js (or removing onclick) could be useful, just so we dont have to remember that we need to add that if we ever were to migrate from bootstrap.